spec/support/spec_config.rb

```diff
@@ -328,6 +328,10 @@ def fle_aws_arn
     ENV['MONGO_RUBY_DRIVER_AWS_ARN']
   end
 
+  def mongocryptd_port
+    ENV['MONGO_RUBY_DRIVER_MONGOCRYPTD_PORT'] || 27020
```

Comment:
`ENV['MONGO_RUBY_DRIVER_MONGOCRYPTD_PORT']` is a string if provided but the default value is an integer.